libs/admin/src/lib/components/workgroups/workgroups.component.html

```diff
@@ -1,31 +1,41 @@
-<div>
-  <button mat-button (click)="createWorkgroup()">Workgroup erstellen</button>
-  @if (selectedWorkgroup || mode === 'create') {
-  <asset-sg-workgroup-edit [workgroup]="selectedWorkgroup" [users]="users" [mode]="mode"></asset-sg-workgroup-edit>
-
-  }
-  <table mat-table [dataSource]="workgroups" class="mat-elevation-z8 table">
+<div class="workgroups">
+  <div class="workgroups__header">
+    <h1 translate>admin.workgroups</h1>
+    <button asset-sg-primary routerLink="create" translate>admin.workgroupPage.create</button>
+  </div>
+  <table mat-table [dataSource]="workgroups" class="mat-elevation-z8 workgroups__table">
     <ng-container matColumnDef="name">
-      <th mat-header-cell *matHeaderCellDef>Name</th>
+      <th mat-header-cell *matHeaderCellDef translate>admin.name</th>
       <td mat-cell *matCellDef="let workgroup">{{ workgroup.name }}</td>
     </ng-container>
-    <ng-container matColumnDef="status">
-      <th mat-header-cell *matHeaderCellDef>Deaktiviert</th>
-      <td mat-cell *matCellDef="let workgroup">{{ workgroup.disabled_at ? workgroup.disabled_at : "-" }}</td>
-    </ng-container>
     <ng-container matColumnDef="users">
-      <th mat-header-cell *matHeaderCellDef>Anzahl Benutzer</th>
-      <td mat-cell *matCellDef="let workgroup">{{ workgroup.users.length }}</td>
+      <th mat-header-cell *matHeaderCellDef translate>admin.users</th>
+      <td mat-cell *matCellDef="let workgroup">
+        @for (user of workgroup.users; track user.userId) {
+        <span>{{ user.user.email }}, </span>
+        }
+      </td>
+    </ng-container>
+    <ng-container matColumnDef="status">
+      <th mat-header-cell *matHeaderCellDef translate>admin.workgroupPage.isActive</th>
+      <td mat-cell *matCellDef="let workgroup">{{ workgroup.disabled_at ? "inaktiv" : "aktiv" }}</td>
     </ng-container>
+
     <ng-container matColumnDef="actions">
-      <th mat-header-cell *matHeaderCellDef>Aktionen</th>
+      <th mat-header-cell *matHeaderCellDef translate>admin.actions</th>
       <td mat-cell *matCellDef="let workgroup">
-        <button mat-button (click)="selectWorkgroup(workgroup)">Editieren</button>
-        <button mat-flat-button>Benutzer hinzufügen</button>
-        <button mat-icon-button (click)="deactivateWorkgroup(workgroup)">Deaktivieren</button>
+        <div class="workgroups__table__actions">
```

Comment:
Do not offer more than one action in the column but a `...` menu instead.
Also make sure that the last few columns dont scale in size but remain small - the `User` column is the one to take the space